README.md

```diff
@@ -0,0 +1,29 @@
+# SciGuides
```

Comment:
Add something about this being a living document. I know it's implied in various places, but I think it deserves to be explicitly stated